package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "omb-eregs",
+  "version": "1.0.0",
+  "description": "This repository contains code necessary to run the White House Office of Management and Budget (OMB) instance of [eRegulations](https://eregs.github.io) (a regulation parser, API, and viewer).",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack --config mywebpack.config.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/18F/omb-eregs.git"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Another for later -- we probably want to use author/license values. [Current recommendations](https://github.com/18F/18f-cli/blob/develop/npm/index.js#L24-L25) are "18f" and "CC0-1.0"